src/core/pages/PatientSummary/SettingsDialog.js

```diff
@@ -27,7 +27,7 @@ export default class SettingsDialog extends Component {
         const { open } = this.state;
         return (
             <div>
-                <IconButton aria-haspopup="true" color="inherit" onClick={() => this.toggleDialog()}>
+                <IconButton id="icon-settings" aria-haspopup="true" color="inherit" onClick={() => this.toggleDialog()}>
```

Comment:
why do you need an id here?

Reply:
This id is necessary for User Tour. But I forgot to add this id in previous PR.